examples/complete/main.tf

```diff
@@ -117,7 +117,9 @@ module "cos_bucket1" {
   existing_key_protect_instance_guid = module.key_protect_all_inclusive.key_protect_guid
   key_protect_key_crn                = module.key_protect_all_inclusive.keys["${local.key_ring_name}.${local.key_name}"].crn
   sysdig_crn                         = module.observability_instances.sysdig_crn
-  activity_tracker_crn               = local.at_crn
+  # disable retention for test environments - enable for stage/prod
+  retention_enabled    = false
```

Comment:
Remove the line from the comment above the bucket definition that says it has retention as well.